docs/TROUBLESHOOTING.md

```diff
@@ -90,4 +90,12 @@ There is also [a feature request on Mountpoint](https://github.com/awslabs/mount
 ## I'm using an S3 Outposts bucket and am getting 'The bucket does not exist' errors
 
 When using S3 Outposts, it is required to include the full ARN for your Outpost bucket in the `bucketName` field.
-See [the S3 Outposts example](../examples/kubernetes/static_provisioning/outpost_bucket.yaml).
\ No newline at end of file
+See [the S3 Outposts example](../examples/kubernetes/static_provisioning/outpost_bucket.yaml).
+
+## My Pod is stuck at `ContainerCreating` with error "driver name s3.csi.aws.com not found in the list of registered CSI drivers"
```

Comment:
Might also worth mentioning this in `CONFIGURATION.md` and adding a changelog entry

Reply:
We should probably add a section on `CONFIGURATION.md` that's explaining this feature and enabling it. We should then link to that section from here.